mmv1/third_party/terraform/transport/config.go.tmpl

```diff
@@ -558,10 +629,10 @@ func (c *Config) synchronousTimeout() time.Duration {
 }
 
 // Print Identities executing terraform API Calls.
-func (c *Config) logGoogleIdentities() error {
+func (c *Config) logGoogleIdentities(ctx context.Context) error {
 	if c.ImpersonateServiceAccount == "" {
 
-		tokenSource, err := c.getTokenSource(c.Scopes, true)
+	tokenSource, err := c.getTokenSource(ctx, c.Scopes, false)
```

Comment:
Was it intentional to change true -> false here?

Reply:
reverted https://github.com/GoogleCloudPlatform/magic-modules/pull/12714/commits/7d01867b46bd64ec96a16eab526b32dc126a138b